__tests__/news/news.mutation.test.js

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+
+let NEWS_ID = '';
+const NEWS_DOES_NOT_EXIST_ID = '1f2ad470eb08183384e6111b';
+describe('test news mutations', () => {
+  test('#1 should add news to database', async () => {
+    const res = await client
+      .mutate({
+        mutation: gql`
+          mutation {
+            addNews(
+              news: {
```

Comment:
Please, use `$variables` for the passing data inside the mutation instead of its direct hardcoding in the query string. It is much more convenient and error-prone and also allows you to have all of your data in some separate object that you can use for the assertions with the data that you get back from server

Reply:
fixed